front/src/utils/date.ts

```diff
@@ -138,3 +140,14 @@ export function convertUTCDateToLocalDate(date: number) {
   const timeDifferenceMinutes = new Date().getTimezoneOffset();
   return Math.abs(timeDifferenceMinutes) * 60 + date;
 }
+
+export function convertIsoUtcToLocalTime(isoUtcString: string): string {
+  const date = new Date(isoUtcString);
+  const offset = date.getTimezoneOffset() * 60000;
+  const localTime = new Date(date.getTime() - offset);
+  return localTime.toISOString();
```

Comment:
Hmm. This will return a `Date` with the `Z` suffix, but the time isn't actually in UTC timezone… Is this really what we want here?

Maybe we can use dayjs here instead?

```js
return dayjs(isoUtcString).local().format()
```

Reply:
agreed